packages/liveblocks-node/src/client.ts

```diff
@@ -299,21 +736,27 @@ export class Liveblocks {
   }): Promise<CommentData> {
     const { roomId, threadId, commentId } = params;
 
-    const resp = await this.get(
-      `/v2/rooms/${encodeURIComponent(roomId)}/threads/${encodeURIComponent(
-        threadId
-      )}/comments/${encodeURIComponent(commentId)}`
+    const res = await this.get(
+      url`/v2/rooms/${roomId}/threads/${threadId}/comments/${commentId}`
     );
+    return (await res.json()) as Promise<CommentData>;
+  }
 
-    const body = await (resp.json() as Promise<CommentData>);
+  /**
+   * Checks whether the error is an HttpError.
+   * @param error The error to check.
+   * @returns Whether the error is an HttpError.
+   */
+  public isHttpError(error: unknown): error is HttpError {
+    return error instanceof HttpError;
+  }
```

Comment:
Renamed `ApiError` to `HttpError` and created this helper method inside the class to check if the error is `HttpError`. Additionally, `HttpError` isn't exported as a part of the `@liveblocks/node` package @nvie